lib/src/layer/polyline_layer/painter.dart

```diff
@@ -280,6 +288,9 @@ class _PolylinePainter<R extends Object> extends CustomPainter {
   LatLng _unproject(DoublePoint p0) =>
       camera.crs.projection.unprojectXY(p0.x, p0.y);
 
+  // TODO: Fix bug where wrapping layer in some widgets (eg. opacity) causes the
+  // features to not move unless this is `true`, but `true` significantly impacts
+  // performance
```

Comment:
Same thoughts as for the PolygonLayer, but seeing this comment here i assume it's unrelated to drawVertices?

Reply:
See above.